src/qililab/experiment/base_experiment.py

```diff
@@ -47,7 +45,6 @@ class BaseExperiment(ABC):
     execution_manager: ExecutionManager
     results: Results
     results_path: Path | None
-    _plot: LivePlot | None
     _remote_id: int
```

Comment:
Remove `_remote_id` aswell

Reply:
This is still pending, unresolving comment :)

And I guess you will also need to edit the `test_base_experiment` and `test_experiment`, search for `_remote_id` in VSCode search interface 🙌

Reply:
Addressed, resolving it!

Reply:
Missing the ones in `test_base_experiment`, commiting it myself!